src/core/ext/filters/client_channel/backup_poller.cc

```diff
@@ -61,19 +61,10 @@ static backup_poller* g_poller = nullptr;  // guarded by g_poller_mu
 static grpc_core::Duration g_poll_interval =
     grpc_core::Duration::Milliseconds(DEFAULT_POLL_INTERVAL_MS);
 
-GPR_GLOBAL_CONFIG_DEFINE_INT32(
```

Comment:
One drawback of centralizing the definition of these config params is that it will be less obvious when there are no longer any users of a given parameter and it can be removed.  Just as one example, prior to this PR, when we eventually remove this backup poller, this parameter would have been removed at the same time; with this PR, it might be easier to forget.  Is there a reasonable way we can have some automated check for this kind of thing?

This isn't a blocker, but would be good to add some check for this if we have a reasonable way to do so.